lib/output.js

```diff
@@ -597,6 +598,12 @@ function OutputStream(options) {
             return true;
         }
 
+        if (output.option('webkit')
+            && output.parent(0) instanceof AST_PropAccess
+            && output.parent(1) instanceof AST_Assign) {
```

Comment:
Yup, `phantomjs 2.5.0-beta` got some new `webkit` bug - the current release version works just fine.